lib/screens/transfer/send/send_confirmation/interactor/usecases/send_transaction_use_case.dart

```diff
@@ -33,6 +33,7 @@ class SendTransactionUseCase {
         return ValueResult(SendTransactionResponse(transactionModel: transactionModel, profiles: profiles));
       }
     }).catchError((error) {
```

Comment:
This debug printout is crucial, otherwise when debugging and something goes wrong in the transaction, we have no idea what happened.